app/EvidencioAPI.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace App;
+
+class EvidencioAPI
+{
+  /*
+   * Helper method to perform an API call
+   * @param $path URL to the call without leading "https://www.evidencio.com/api/"
+   * @param $params list of parameters as a key-value array
+   * @return API response converted into a PHP variable
+   * @throws JsonDecodeException if JSON response could not be decoded
+   * Other possible exceptions are listed in Guzzle's documentation
+   */
+  private static function fetch($path, $params = [])
+  {
+    $client = new \GuzzleHttp\Client(['base_uri' => 'https://www.evidencio.com/api/']);
+    $res = $client->post($path,
+    [
+      'headers' => [
+        'Accept' => 'application/json',
+        'Authorization' => config('app.evidencio_key')
+      ],
+      'form_params' => $params
+    ]);
+
+    $json = json_decode($res->getBody(),true);
+
+    if(is_null($json))
+    {
+      throw new Exceptions\JsonDecodeException($res->getBody());
```

Comment:
For readability it is nicer if you could reference `JsonDecodeException` directly by including its namespace. e.g. `Use Exceptions\JsonDecodeException`